packages/components/src/components/icon/icon.jsx

```diff
@@ -15,6 +15,7 @@ const Icon = React.forwardRef(
             icon,
             id,
             onClick,
+            onMouseDown,
```

Comment:
added onMouseDown as it would be prioritised in the event queue over onBlur within autocomplete